### PR TITLE
NAS-119051 / 22.12 / Fix `as any` casts

### PR DESCRIPTION
**Pull request by bvasilenko**
**Summary**

1st part of this ticket is a problem on middleware side.

2nd part is removing `any` casts, and it's done here:

### CODE REVIEW

**Comment by undsoft** [comment]
Soo, we're saying that we expect `passphrase2` field to be present in `certificate.query`?
That's kinda weird, let's wait for middleware confirmation that they expect those fields there before merging.

**Review by AlexKarpov98 — APPROVED** [review]
`as any` cast is removed now

**Pull request merged**
